units/src/fee_rate/mod.rs

```diff
@@ -81,6 +86,77 @@ impl FeeRate {
         (self.to_sat_per_kwu() + (1000 / 4 - 1)) / (1000 / 4)
     }
 
+    /// Displays the fee rate using sat/kwu unit.
+    ///
+    /// This unit is precise, so it doesn't need rounding options.
+    ///
+    /// The unit is displayed by default, call [`without_unit`](Display::without_unit) on the
+    /// returned value to hide it.
+    pub fn display_sat_per_kwu(self) -> Display {
+        Display {
+            fee_rate: self,
+            format: Format::SatPerKwu,
+            display_unit: true,
+        }
+    }
+
+    /// Displays the fee rate using sat/kwu unit, rounding if needed.
+    ///
+    /// By default this has precision of 3 decimal places (but the trailing zeros are not
+    /// displayed). In that case the value is precise and no rounding is applied.
+    ///
+    /// However, using smaller formatter precision will require rounding of the numer. This method
```

Comment:
```suggestion
    /// However, using smaller formatter precision will require rounding of the nubmer. This method
```

Reply:
And in a bunch of other places too.

Reply:
your correction is misspelled heh: s/nubmer/number